fiftyone/utils/torch.py

```
@@ -1125,6 +1239,332 @@ def __call__(self, img):
+    """
+    if img_np.ndim == 2:
+        return np.repeat(img_np[..., None], 3, axis=2)
+    elif img_np.shape[2] == 1:
+        return np.repeat(img_np, 3, axis=2)
+    elif img_np.shape[2] == 4:
+        return img_np[..., :3]
+    return img_np
+
+
+def convert_to_tensor_chw(
+    img: Union[Image.Image, np.ndarray, torch.Tensor]
```

Ruff (0.14.8)

Use `X | Y` for type annotations

Convert to `X | Y`

(UP007)